code/index.html

```diff
@@ -17,7 +17,7 @@
       <img
         class="guess-who-icon"
         src="images/guess-who-bubble.png"
-        alt="Guess Who"
+        alt="Guess Who?"
```

Comment:
Using "Guess Who Icon" instead could also add more descriptiveness to the alt tag if the img did not display